gradle-plugin/it/src/it/openliberty/build.gradle

```diff
@@ -0,0 +1,64 @@
+plugins {
+    id 'war'
+    id 'io.openliberty.tools.gradle.Liberty' version '3.3.1'
```

Comment:
Shouldn't this plugin require the liberty feature configuration block?

In the adoc you specify (for XML) that these settings should be the plugin config:
```xml
    <featureManager>
        <feature>mpHealth-4.1</feature>
    </featureManager>
```